src/components/core/geoChart/GeoChartRenderer.tsx

```diff
@@ -148,8 +163,8 @@ export default class GeoChartRenderer extends React.Component<IGeoChartRendererP
         } = this.props;
```

Comment:
It could be
```
        const { props, chart } = this;
        const {
            config: { selectedSegmentItems },
            geoData,
        } = props;
```

Reply:
updated.
thanks